### ISSUE TITLE
[Northrend] [Quest] Shadow Vault Area / It's All Fun and Games

### ISSUE THREAD

**Issue by Cry0tek** [dalaranwow/dalaran-wow#3903]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (OLD - ... tags should no longer be used)
[//]: # (Priority - ... tags are assigned by team members)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Description**: Game freezes and characters get stuck

**Current behaviour**: Flying anywhere near the Ocular area in Shadow Vault to do the quest: "It's All Fun and Games" freezes the game and gets the character stuck. Reloading or relogging does not solve the issue and you would need to log in on an alt to open a ticket to get a GM to unstuck the other toon.

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
DB/Quest: If He Cannot Be Turned - Despawn all affected creatures 5 minutes after summon and make the summonobject only usable every 5 minutes

* also increase HP-Pool for Vile, Lady Nightswood and he Leaper
* closes https://github.com/dalaranwow/dalaran-wow/issues/3903
* commit: 99b1f40fd78ab9058023f43d062e59fc7140b9f1

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by TeridanDW** [comment]
Works as intended on PTR.

Note:

>  - also increase HP-Pool for Vile, Lady Nightswood and he Leaper

to 2 mil hp each seems a bit too much for me.
